tests/PHPUnit/System/expected/test_CustomEvents__Live.getLastVisitsDetails_day.xml

```diff
@@ -794,8 +794,8 @@
 				<eventCategory>Music</eventCategory>
 				<eventAction>play50%</eventAction>
 				<bandwidth />
-				<timeSpent>60</timeSpent>
-				<timeSpentPretty>1 min 0s</timeSpentPretty>
+				<timeSpent>0</timeSpent>
```

Comment:
Couldn't figure out where these changes are from. The tests seem to pass in 3.x-dev. 

Not sure how this would change in this PR since in the tests itself the fingerprintSalt is basically not set and nothing changes.

Reply:
Looking at the tests it seems the changes to the test files can be reverted.